definitions/infra-gcprouternatgateway/summary_metrics.yml

```
@@ -0,0 +1,33 @@
+newConnections:
+  title: New connections
+  UNIT: COUNT
```

```suggestion
  unit: COUNT
```

I think there are several files with this. Remember you can run the linter to verify these things
https://github.com/newrelic-experimental/entity-synthesis-definitions/tree/main/validator#local-setup

---

Fixed!